### DISCUSSION TITLE
Can we have "draft mode"  for discussions?

### DISCUSSION THREAD

**Discussion by dubsalot** [community/community#36989] [Discussions] [answered]
# Apologies!
This topic has been brought up many times. My sEaRcHiNg sKiLlz on mobile failed me!

## Context
My employer is slowly ramping up GitHub EMU so more and more associates are starting to see it/use it. I use Discussions in my Org mainly to Show & Tell the cool things I've been working on, and also evangelizing GitHub quite a bit. I like to polish my Discussion topics with screenshots, links to live examples and repos. Sometimes, I'd like to hit the pause button and finish the next day.

## Request Draft/Unpublished State
Maybe I am just overlooking it, but I find myself wanting a "save draft" option, and often. I put a good deal of work into the presentations, and worry if I don't "Start Discussion", I risk losing this work.  I would like to keep a presentation/show&tell in draft state while I proof it and finalize it. When done, I would like to "publish it" for all to see in my org.

Is this already an option? If not, can I request it?

Maybe I'm thinking about it wrong. 'm open to any feedback you have.

**Comment by ossanna16** [comment, marked as answer]
Hi there @dubsalot and welcome to our community 😄. Thanks so much for the suggestion!

**Reply by LitalFiverr** [reply]
Is it going to be supported?

**Reply by girlyguppy** [reply]
Please add this. I just had the page refresh and it deleted a whole paragraph
